content/chainguard/chainguard-enforce/events/create-github-issues/index.md

```diff
@@ -1,15 +1,15 @@
 ---
-title: "Create GitHub Issues from Enforce Cloud Events"
+title: "Create GitHub Issues from Enforce CloudEvents"
 type: "article"
-description: "Creating GitHub Issues from Enforce Cloud Events"
+description: "Creating GitHub Issues from Enforce CloudEvents"
 date: 2023-04-11T15:22:20+01:00
 lastmod: 2023-04-11T15:22:20+01:00
 draft: false
 tags: ["Enforce", "Product", "CloudEvents"]
 images: []
 menu:
   docs:
-    parent: "chainguard-enforce"
+    parent: "cloudevents"
 weight: 650
```

Comment:
```suggestion
weight: 10
```

I'd like us to be a bit more conscious and deliberate about how we order things (now that everything is in its own place)

Reply:
I like it, instead of throwing darts, everything has its right place